packages/material-ui-based-components/src/components/DropDown.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback } from 'react';
+import { withRuleEngine } from '../shared/withRuleEngine';
+import { PROPS } from '../utils/types';
+import { FormControl, FormHelperText, InputLabel, MenuItem, Select } from '@mui/material';
+
+const DropDownComponent = (props: PROPS) => {
+  const {
+    label, required, enumNames, enum: enums, enabled, id,
+    isError, description, errorMessage, name, value, type
+  } = props;
+
+  const changeHandler = useCallback((event: any) => {
+    props.dispatchChange(event.target.value);
+  }, [props.dispatchChange]);
+
+  const blurHandler = useCallback((event: any) => {
+    props.dispatchBlur(event.target.value);
+  }, [props.dispatchBlur]);
+
+  const focusHandler = useCallback((event: any) => {
+    props.dispatchFocus(event.target.value);
+  }, [props.dispatchFocus]);
+
+  let options = [{ value: enums ? enums[0] : '', label: enumNames ? enumNames[0] : enums ? enums[0] : '' }];
+
+  if (enums) {
+    for (let i = 1; i < enums.length; i++) {
+      let ob = { value: enums[i], label: enumNames ? enumNames[i] : enums[i] };
+      options = [...options, ob];
+    }
+  }
+
+  return (
+    <FormControl
+      variant="outlined"
```

Comment:
variant should be dynamic. It will come from the props.properties['afs:layout'].variant and default value of this outlined

Reply:
implementing dynamic variant